Dockerfile

```diff
@@ -11,7 +11,29 @@ ENV NEW_HEARTBEAT_ENDPOINT=$NEW_HEARTBEAT_ENDPOINT
 ENV APP_VERSION=$APP_VERSION
 LABEL maintainer="mandarons"
 
+# Set environment variables
 ENV HOME="/app"
+ENV PUID=911
+ENV PGID=911
+
+# Install system dependencies and create user first for better caching
+RUN \
+  echo "**** update package repository ****" && \
+  apk update && \
+  echo "**** install packages ****" && \
+  apk add --no-cache \
+    python3 \
```

Comment:
May not need to install python due to python:3.10-alpine3.22 base image that comes with python 3.10. Verify this.